src/main/java/com/vineela/dummy/emp/api/controller/HelloController.java

```diff
@@ -0,0 +1,27 @@
+package com.vineela.dummy.emp.api.controller;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1")
```

Comment:
please don't hard code API endpoint /api/v1, instead of hard coding move this to constants

Reply:
I have taken constants for /api/v1 and committed the code . please review